.pre-commit-config.yaml

```diff
@@ -0,0 +1,61 @@
+repos:
+  # other fast helpful checks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-yaml
+        args: [--allow-multiple-documents]
+      - id: check-json
+      # - id: end-of-file-fixer
+      # - id: trailing-whitespace
+      #   exclude: ^manifests/gcp_marketplace/test/*
+      # - id: debug-statements
+      # - id: check-merge-conflict
+      # - id: name-tests-test
+      # - id: double-quote-string-fixer
+      # - id: no-commit-to-branch
+      #   args: [--branch, master]
```

Comment:
Maybe we can remove these check for now, and add them if we need to use them with Katib.

Reply:
Done!